<?xml version="1.0"?>

<launch>
  <arg name="ns" default="robot"/>

  <group ns="$(arg ns)">
    <node name="main_control" pkg="quadrotor_tunnel_nav" type="main_control_node" args="wait" />
    <node name="obstacle_avoidance" pkg="quadrotor_tunnel_nav" type="obs_avoid_node" args="" />
    <node name="altitude_adjustment" pkg="quadrotor_tunnel_nav" type="altitude_node" args="" />
    <node name="turning_around" pkg="quadrotor_tunnel_nav" type="turn_node" args="" />
    <node name="steering" pkg="quadrotor_tunnel_nav" type="steer_node" args="" />
    <node name="staying_at_the_right_side" pkg="quadrotor_tunnel_nav" type="right_wall_node" args="" />
    <node name="keep_going" pkg="quadrotor_tunnel_nav" type="keep_going_node" args="" />
  </group>

</launch>
